modules/nordic/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+if(CONFIG_HAS_NORDIC_DRIVERS OR CONFIG_HAS_NRFX)
+  zephyr_library()
+endif()
+
+if(CONFIG_HAS_NORDIC_DRIVERS)
+  add_subdirectory(${ZEPHYR_NORDIC_MODULE_DIR}/drivers drivers)
```

You can use the `ZEPHYR_CURRENT_MODULE_DIR` variable.
That variable is intended for internal module code, and the `ZEPHYR_<NAME>_MODULE_DIR` is intended when referring to other modules.

```suggestion
   add_subdirectory(${ZEPHYR_CURRENT_MODULE_DIR}/drivers drivers)
```

---

Thanks for the hint. I wasn't aware of that. I'll certainly switch to using this variable in the upcoming update of the PR.